packages/components/src/toolbar/toolbar/style.scss

```diff
@@ -18,6 +18,16 @@
 	}
 }
 
+// Make the spacing consistent between controls in the vertical toolbar, only in zoom out.
+.zoom-out-toolbar-popover {
+	.components-accessible-toolbar,
+	.components-toolbar {
+		.components-button {
+			height: $block-vertical-toolbar-height;
```

Comment:
Is there a clearer way to do this?

Reply:
We can use `$grid-unit-50` from [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_variables.scss#L37) variable instead of creating a new one for `40px`. Should I go ahead with that?

Reply:
Yes please reuse an existing variable or hard code for the timebeing.

Reply:
Removed the variable and used hard code value.